onnx/version_converter/adapters/pad_10_11.h

```diff
@@ -27,15 +27,17 @@ class Pad_10_11 final : public Adapter {
       node->addInput(v_pads);
       node->removeAttribute(kpads);
       // Turn value attribute into input
-      Tensor t_value;
-      t_value.elem_type() = TensorProto_DataType_FLOAT;
-      auto& data_value = t_value.floats();
-      data_value.emplace_back(node->f(kvalue));
-      Node* constant = graph->create(kConstant);
-      constant->insertBefore(node);
-      constant->t_(kvalue, t_value);        
-      node->addInput(constant->output());
-      node->removeAttribute(kvalue);
+      if (!node->hasAttribute(kmode) || node->s(kmode) == "constant") {
```

Comment:
should this be node->hasAttribute(kmode) && || node->s(kmode) == "constant"
Can you elaborate a little more about this check. Thanks!

Reply:
the default mode is 'constant' and _node->hasAttribute(kmode)_ is false
so, if **node->hasAttribute(kmode) == false**, it is a DEFAULT 'constant' mode

Reply:
I wonder if the IR automatically takes care of returning the default-value? If not, then shouldn't we also check if the "value" attribute is present down below, before using?